builder/vsphere/supervisor/step_create_source.go

```diff
@@ -206,6 +204,12 @@ func (s *StepCreateSource) initStep(state multistep.StateBag) error {
 		kubeClient client.Client
 	)
 
+	if s.Config.ImageName == "" {
+		if s.Config.ImageName, ok = state.Get(StateKeyImportedImageName).(string); !ok {
+			return fmt.Errorf("the image name should be specified in config 'image_name' or generated from image import")
+		}
+	}
```

Comment:
Can we also check if both `s.Config.ImageName` and `state.Get(StateKeyImportedImageName)` are set, and output a warning that the imported image will be skipped in this case?

Reply:
Agree, it would be helpful to show a message in the output.